### DISCUSSION TITLE
文章更新时间的问题

### DISCUSSION THREAD

**Discussion by 9201314** [HEIGE-PCloud/DoIt#343] [Q&A] [answered]
文章更新时间总是和文章创建时间一致，问一下在哪里可以配置？
![image](https://user-images.githubusercontent.com/45234281/140710294-808e76a3-6853-4ebc-83b2-d20e38d64f32.png)

**Comment by HEIGE-PCloud** [comment, marked as answer]
https://hugodoit.pages.dev/zh-cn/theme-documentation-content/#front-matter

通过配置 front matter 中 lastmod 值来手动修改更新时间。

https://gohugo.io/getting-started/configuration/#enablegitinfo

通过开启 enableGitInfo 来让 Hugo 自动设置更新时间。
